src/Peddler/DoubleGenerator.cs

```diff
@@ -0,0 +1,237 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Peddler {
+    /// <summary>
+    ///   An implementation of <see cref="IDistinctGenerator{Double}" />.
+    /// </summary>
+    /// <inheritdoc />
```

Comment:
I've never seen `<inheritdoc/>` in conjunction with overriding documentation. Is that valid?

Reply:
This looks to be fine ([source](http://tunnelvisionlabs.github.io/SHFB/docs-master/SandcastleBuilder/html/79897974-ffc9-4b84-91a5-e50c66a0221d.htm)):
>Using it in conjunction with other tags allows you to inherit common information such as value and parameter descriptions while overriding the inherited documentation for other tags such as `<summary>` and `<remarks>`.